Fixed deleting Automatic Keyword Group in the UI (#9955)
JabRef/jabref: CHANGELOG.md

- We fixed an issue where the encoding header in a bib file was not respected when the file contained a BOM (Byte Order Mark). [#9926](https://github.com/JabRef/jabref/issues/9926)
- We fixed an issue where cli help output for import and export format was inconsistent. [koppor#429](https://github.com/koppor/jabref/issues/429)
- We fixed an issue where no preview could be generated for some entry types and led to an exception [#9947](https://github.com/JabRef/jabref/issues/9947)

> - We fixed an issue is wa delete an Automatic Keyword Group in the UI[#9778](https://github.com/JabRef/jabref/issues/9778)

Reviewer: ```suggestion
- We fixed an issue where an Automatic Keyword Group could not be deleted in the UI. [#9778](https://github.com/JabRef/jabref/issues/9778)
```
